src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    public Ball() {
+        color = new ColorSupplier().getRandomColor();
+        number = new Random().nextInt(99) + 1;
+    }
```

Comment:
remove this constructor and create a new one where you can pass color and number